xrmocap/core/evaluation/mvp_evaluation.py

```diff
@@ -1,67 +1,341 @@
+# yapf: disable
+import logging
 import numpy as np
+import os
+import time
 import torch
-from typing import List, Tuple
+from prettytable import PrettyTable
+from torch.utils.data import DataLoader
+from typing import List, Tuple, Union
+from xrprimer.utils.log_utils import get_logger
 
-from xrmocap.data.dataset.base_dataset import BaseDataset
+from xrmocap.core.evaluation.align_keypoints3d import align_keypoints3d
+from xrmocap.data_structure.keypoints import Keypoints
+from xrmocap.model.architecture.base_architecture import BaseArchitecture
+from xrmocap.utils.distribute_utils import collect_results, is_main_process
+from xrmocap.utils.mvp_utils import (
+    AverageMeter, convert_result_to_kps, norm2absolute,
+)
+from .metrics import calc_limbs_accuracy, evaluate
```

Comment:
The name `evaluate` is a little general, which contains mpjpe, pampjpe and pck at the same time. Any better name to describe it?

Reply:
After offline discussion, it will be left to #50